src/sentry/static/sentry/app/views/alerts/rules/details/body.tsx

```diff
@@ -308,48 +256,31 @@ export default class DetailsBody extends React.Component<Props> {
             </PanelBody>
             {this.renderChartActions()}
           </ChartPanel>
-          <DetailWrapper>
-            <ActivityWrapper>
-              <StyledNavTabs underlined>
-                <li className="active">
-                  <Link to="">{t('Activity')}</Link>
-                </li>
-
-                <SeenByTab>
-                  {incident && (
-                    <StyledSeenByList
-                      iconPosition="right"
-                      seenBy={incident.seenBy}
-                      iconTooltip={t('People who have viewed this alert')}
-                    />
-                  )}
-                </SeenByTab>
-              </StyledNavTabs>
-              <Activity
-                incident={incident}
-                params={params}
-                incidentStatus={!!incident ? incident.status : null}
-              />
-            </ActivityWrapper>
-          </DetailWrapper>
         </Layout.Main>
         <Layout.Side>
+          <ChartParameters>
+            {tct('Metric: [metric] over [window]', {
+              metric: <code>{rule?.aggregate ?? '\u2026'}</code>,
+              window: (
+                <code>
+                  {rule?.timeWindow ? (
+                    <Duration seconds={rule?.timeWindow * 60} />
+                  ) : (
+                    '\u2026'
+                  )}
+                </code>
+              ),
+            })}
+            {(rule?.query || rule?.dataset) &&
+              tct('Filter: [datasetType] [filter]', {
```

Comment:
![image](https://user-images.githubusercontent.com/9372512/105536214-d7368b00-5ca4-11eb-9f23-4db082c520cd.png)

I think this css got a bit messed up under `Filter` there is a misplaced separator line.

but I understand this will get overhauled soon so it's probably not an issue

Reply:
Yes, I'll address the alert rule in the sidebar in a separate pr.